systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -138,10 +148,49 @@ void testTieredStorageWithAivenPlugin() {
         resourceManager.createResourceWithWait(clients.producerStrimzi());
 
         SetupMinio.waitForDataInMinio(suiteStorage.getNamespaceName(), BUCKET_NAME);
+
+        // Create admin-client to check offsets
+        resourceManager.createResourceWithWait(
+            AdminClientTemplates.plainAdminClient(
+                testStorage.getNamespaceName(),
+                testStorage.getAdminName(),
+                KafkaResources.plainBootstrapAddress(testStorage.getClusterName())
+            )
+            .editSpec()
+                .editOrNewTemplate()
+                    .editSpec()
+                        .editFirstContainer()
+                            // TODO - use latest test-clients after https://github.com/strimzi/test-clients/pull/103
+                            .withImage("quay.io/jstejska/test-clients:admin-kafka-3.7.1")
+                        .endContainer()
+                    .endSpec()
+                .endTemplate()
+            .endSpec().build()
+        );
+        final AdminClient adminClient = AdminClientUtils.getConfiguredAdminClient(testStorage.getNamespaceName(), testStorage.getAdminName());
+        // Fetch latest offsets - number of sent messages
+        String offsetData = adminClient.fetchOffsets(testStorage.getTopicName(), "latest");
+        long latestOffset = AdminClientUtils.getPartitionsOffset(offsetData, "0");
+
+        // Fetch earliest-local offsets - number of messages stored locally
+        offsetData = adminClient.fetchOffsets(testStorage.getTopicName(), "earliest-local");
+        long earliestLocalOffset = AdminClientUtils.getPartitionsOffset(offsetData, "0");
+        // Check that data are not present locally, earliest-local offset should be lower than latest offset
+        // latest should be 10000
+        // earliest-local should be lower than 10000
+        assertThat(earliestLocalOffset, lessThan(latestOffset));
```

Comment:
I think this is not what we want. With the current assertion, if the local log is not deleted, the earliest local log offset will be 0, and latestOffset will be 10000, then this assertion is still passed. 
To verify the local log is deleted, we can do:
`assertThat(earliestLocalOffset, greaterThan(0);` 
So that we know the log is deleted, and the earlier local offset is increased.

Note: The local log deletion might not happen immediately, it'd be better to use `waitForLocalLogDeletion` method to wrap it.

Reply:
I updated the logic